app/db/migrations/20210303183552-remove-twitter-auth.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
```

Comment:
mainly curious, where did the need come from to wrap these in transactions? I did a bit of looking around and it seems like this is a good idea, but it might be good to leave a comment on why and whether we'd want this to be best practice for further migrations removing fields

Reply:
The truth is I don't know if I can explain with confidence what transactions are. With my fairly limited understanding of SQL my understanding is that these are "safer" ways of doing multiple (batch) actions on a database, which treats multiple actions as one "atomic" action rather than a sequence of individual actions. You can let me know if this understanding is incorrect.

And the reality is that I looked around the codebase for a way we did this in the past, and we used transactions here:
https://github.com/streetmix/streetmix/blob/main/app/db/migrations/20200705103525-update-vote.js

At the moment I don't have the confidence to say this is "the" best practice, only that if we were to adopt a pattern, this is one we currently employ.

Reply:
Sounds good. Postgres is def an [ACID compliant ](https://en.wikipedia.org/wiki/ACID )database, but it may be that Sequelize itself is not? So its possible if this migration ran, each individual row would be successfully affected or rolled back, but the migration as a whole could error out at some point and then the ORM might be in a bit of a Catch-22. 

More discussion on that is here: https://github.com/sequelize/sequelize/issues/7902